requirements/docs.txt

```diff
@@ -1,9 +1,9 @@
 # dependencies for building docs, separate from dev.txt as this is also used for builds on readthedocs.org
 # core dependencies
 sphinx>=4.2.0, <5.0.0
-sphinx-autodoc-typehints>=1.12.0, <2.0.0
-sphinx-autoapi>=1.8.0, <2.0.0
-sphinx-rtd-theme>=1.0.0, <2.0.0
+sphinx-autodoc-typehints>=1.12.0, <=2.0.0
+sphinx-rtd-theme>=1.0.0, <=2.0.0
```

Comment:
Strict upper bounds for these two should be upheld as equality was a mistake/typo.